CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+Change the name of `BranchMutMappedMut` to `MappedBranchMut`
```

Comment:
Add `- ` at the beginning of the list item (see https://keepachangelog.com/en/1.0.0/)